tests/functional/validator/conftest.py

```diff
@@ -30,6 +30,34 @@ def schema():
     }
 
 
+@pytest.fixture
+def schema_default():
+    return {
+        "$schema": "http://json-schema.org/draft-07/schema",
+        "$id": "http://example.com/example.json",
```

Comment:
Sonarcloud is [complaining](https://sonarcloud.io/project/security_hotspots?id=aws-powertools_powertools-lambda-python&pullRequest=4839&issueStatuses=OPEN,CONFIRMED&sinceLeakPeriod=true) about security here. Can you please make this endpoint https?

 ```suggestion
        "$schema": "https://json-schema.org/draft-07/schema",
        "$id": "https://example.com/example.json",
```